source/appModules/winword.py

```diff
@@ -23,6 +51,13 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 
 
 class WinwordWordDocument(WordDocument):
+	def _get_description(self) -> str:
+		try:
+			curView = self.WinwordWindowObject.view.Type
+			return ViewType(curView).displayString
```

Comment:
If Word adds a description or help text at any point, it'll be thrown away after this PR. Is this desirable?

Reply:
@nvdaes  you could consider appending or prepending the view type to the description. Something like 
```suggestion
			return f"{ViewType(curView).displayString} super()._get_description()"
```